doc: add quick start documentation (#555)
dynamic-framework/dynamic-ui: libraries/dynamic-ui-react/stories/QuickStartWidget.mdx

import { Meta } from '@storybook/blocks';
import { DAlert } from '../src';

<Meta title="Design System/Quick Start/Widget" />

<div className="inline-story">
  # Widget configurations

  <br />
  ### Style in Widget
  <DAlert type="warning" soft>It's recommended to use [the style in the platform](/docs/design-system-quick-start-platform--docs#style-in-modyo-platform)</DAlert>

  Import the framework style before the import of the widget style.

  ```tsx
  import '@dynamic-framework/ui-react/dist/css/dynamic-ui.css';
  ```
  > [Example in dynamic-react-base-template](https://github.com/dynamic-framework/dynamic-react-base-template/blob/fc60dc619b8a25c809d8f0060b0d325d3d5a932c/src/index.tsx)

  <br />
  ### Font in Widget
  > the framework use `Source Sans Pro`

  Configure the font at `public/index.html` file.

  <br />
  ### Icons in Widget
  Configure the icon font at `public/index.html` file.

>   > To configure an icon font with material symbols style settings you will need a special context configuration
>   explained in [Context Provider / Icon Settings](/docs/design-system-components-context-provider--docs#icon-settings)

Reviewer: ```suggestion
  > To configure an icon font with Material Symbols style settings, a special context configuration explained in [Context Provider / Icon Settings](/docs/design-system-components-context-provider--docs#icon-settings) will be required.
```
